XiEditor/AppDelegate.swift

```diff
@@ -369,7 +369,12 @@ class AppDelegate: NSObject, NSApplicationDelegate, XiClient {
         }
     }
 
+    var config: [String: AnyObject] = [:]
```

Comment:
can we call this something like `configCache` or `lastConfig`, and include a brief comment describing the problem that we're working around?

The eventual solution will be to have a separate scope for client config settings that will be unrelated to buffer-specific settings, but this is reasonable for now.

An alternative option would be to _just_ cache these client settings, and handle them separately. That's clearer in its intent, but is a bit more code. I think either approach is fine, but it's good to document our hacks so they're easier to clean up later. :)